### PR TITLE
Update parameters based on macro data

### PR DESCRIPTION
**Pull request by jdebacker**
This PR updates the `default_parameters.json` to values consistent with the Indian economy.

### CODE REVIEW

**Comment by abhishekverma85** [comment]
Value is 5.2% for the period 2000 to 2013

https://labour.gov.in/sites/default/files/Chapter%20-%204.pdf

For g_y_annual.. This is from the Ministry of Labour, Govt of India's report (refer Table 4.1)

**Comment by rickecon** [comment]
@adc2-tpru suggested that `delta_annual` should be 0.049 based on a paper of his. But @indira-tpru said it should be 0.0326.

**Comment by shekharkhetan** [comment]
http://www.mospi.nic.in/sites/default/files/reports_and_publication/statistical_publication/National_Accounts/NAS18/S4.2.pdf
# for government transfers by economic activity , do we need to add the current transfer? Pl clarify in the above link.

**Comment by abhishekverma85** [comment]
for alpha_G : 11.2% (for 2018)

https://data.worldbank.org/indicator/NE.CON.GOVT.ZS?locations=IN

General government final consumption expenditure (% of GDP)

**Comment by dddodums** [comment]
I got a figure on subsidy as a % of GDP in India(subsidy is the only form of transfers in India other than the pensions) thus the following:
Government data shows that subsidy as a percentage of gross domestic product (GDP) has come down to 1.57 percent for 2017-18 and stands at ₹2.34 trillion, one of the lowest since 2008-09, when subsidies formed 2.35 percent of the GDP

http://www.forbesindia.com/article/poll-vault/is-the-budget-subsidy-data-telling-you-the-real-story/52233/1

**Comment by soumyajitrayecon** [comment]
Capital & Labour Income share [Income & Labour share.xlsx](https://github.com/TPRU-India/OG-India/files/3533110/Income.Labour.share.xlsx)
2016-17 
For the Economy: Labour income share = 0.5 & Capital Income Share = 0.5
For the Manufacturing : Labour income share = 0.301 & Capital Income Share = 0.699
For the Service Sector : Labour income share = 0.524 & Capital Income Share = 0.476

**Comment by soumyajitrayecon** [comment]
[Monthly Summary Report of DoE  for July,2019_0.pdf](https://github.com/TPRU-India/OG-India/files/3533151/Monthly.Summary.Report.of.DoE.for.July.2019_0.pdf)

Department of Expenditure report on Government Expenditure July 2019

**Comment by soumyajitrayecon** [comment]
Following r_gov_scale and r_gov_shift were estimated using the attached data.
Period : 1993 to 2018
Government_Securities = 0.81*(Commercial_Paper)+1.16
Central_Government_Securities = 0.82*(Commercial_Paper)+0.93
State_Government_Securities = 0.81*(Commercial_Paper)+1.39

[r_gov_scale & shift.xlsx](https://github.com/TPRU-India/OG-India/files/3533691/r_gov_scale.shift.xlsx)

**Comment by jdebacker** [comment]
@soumyajitrayecon What should I look for in that report?  Can you summarize what that implies for a parameter value?

**Comment by soumyajitrayecon** [comment]
@jdebacker Sir, that report is the expenditure summary of the Government of India for the month of July 2019. This can be used to impute the total spending on government transfers (DBT & others) to households. However further probe needs to be done in order to arrive at a more robust estimate of Expenditure on Transfers/GDP.

**Comment by jdebacker** [comment]
@soumyajitrayecon Thank you for clarifying that for me.  I agree - that report will be useful, but we'll need some additional information.
